Solutions/IronNet IronDefense/Analytic Rules/IronDefense_Detection_Query.yaml

```diff
@@ -49,7 +49,7 @@ customDetails:
   AnalystRating: LogSeverity
   AlertCreatedTime: TimeGenerated
   IronVueUrl: IronVueUrl
-  IronDefenseAlertId: DeviceCustomString4
+  IronDefenseAlertIdentifier: DeviceCustomString4
```

Comment:
Hello @ling-tx please decrease the number of character of this customdetail '**IronDefenseAlertIdentifier**', it should not exceed a limit of 20 characters.